CK3toEU4/Source/EU4World/Country/Country.cpp

```diff
@@ -1108,27 +1108,27 @@ void EU4::Country::assignReforms(const std::shared_ptr<mappers::RegionMapper>& r
 	}
 	// Tsardom
 	else if (details.government == "monarchy" &&
-				(tag == "UKR" || (tag == "RUS" && (orthodoxReligions.count(details.religion) || details.religion == "slavic_pagan" ||
-																  details.religion == "slavic_pagan_reformed"))))
+			 (tag == "UKR" || (tag == "RUS" && (orthodoxReligions.count(details.religion) || details.religion == "slavic_pagan" ||
```

Comment:
I'd restrict tsardom to either orthodox or slavic_pagan_reformed, and details.governmentRank == 3. Makes no sense a large unreformed horde is suddenly a tsardom just because they split from mongolia and have russia as primary tag.

Reply:
I didn't make any changes, I just changed the organization since it was bothering me, although knowing VS, it will likely mess it up again. It doesn't seem to like my styling.

They can only be Russia if they have government rank 3 since it is only mapped to the Empire of Russia, i.e - e_russia.